src/main/resources/application.yml

```
@@ -0,0 +1,10 @@
+spring:
+  application.name: capstone
+  profiles.default: dev
+  config:
+    import:
+      - classpath:config/application-security.yml
+      - classpath:config/application-dev-db.yml
+      - classpath:config/application-gemini.yml
+      - classpath:config/application-clova.yml
+      - classpath:config/application-grpc.yml
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)